### PR TITLE
Adding script to reimage dell servers in Octo lab

### PR DESCRIPTION
**Pull request by AmarnatReddy**
# Description
Adding script to reimage dell servers in Octo lab
`
Usage : python utility/reimage_octo_nodes.py --os_version 8.6 --nodes=cali018,cali021,mero015 --username xxxxx --password xxxxxxx`

We have used kickstart file for doing the reimage.
right now supports os are rhel9.2 and rhel 8.6

Preparing kickstart file doc : 
https://docs.google.com/document/d/11AypHZaZFlaC-rQ0fk0PoIG7m8M_YZ67PW9_T8zDqAE/edit?userstoinvite=uchapaga@redhat.com&role=writer#heading=h.lket79fmpml 

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by neha-gangadhar** [comment]
@AmarnatReddy Looks good to me. Small enhancement may be to add descriptions about the functions and inputs in all the development we do. However since the naming is pretty clear in this case, should be fine.  Please get another review and we can go ahead and merge it.

File: utility/reimage_octo_nodes.py [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+# Define the base URL and credentials
+base_url = "https://{}"
+username = "xxxx"
+password = "xxxx"
+
+# Define a dictionary to store OS versions and their respective ISO URLs
+image_list = {
+    "9.2": "http://10.0.209.99/rhel92ks_updated.iso",
```
Comment (hkadam134) [inline utility/reimage_octo_nodes.py:39]: Can we use magna006 - which is reserved for Baremetal Infra

Comment (AmarnatReddy) [reply utility/reimage_octo_nodes.py]: Installed httpd server on magna006 and updated the iso there.

**Review by hkadam134 — APPROVED** [review]
Looks Good.

**Review by hkadam134 — APPROVED** [review]
Looks Good

**Review by neha-gangadhar — APPROVED** [review]

**Pull request merged**